mathics/builtin/drawing/image.py

```diff
@@ -517,7 +519,7 @@ def apply_resize_width_height(self, image, width, height, evaluation, options):
         h = self._get_image_size_spec(old_h, height)
         if h is None or w is None:
             return evaluation.message(
-                "ImageResize", "imgrssz", Expression(SymbolList, width, height)
+                "ImageResize", "imgrssz", to_mathics_list(width, height)
```

Comment:
Maybe `ListExpression(Integer(width), Integer(height))`?

Reply:
Look at the return signature on `_get_image_spec()`

Reply:
There is some question as to whether we should return optional float. Or possibly whether we should use Real() along with ListExpression().  

However here I want to punt and come back to this at some later stage. And it's probably not worth putting any effort into for this particular place because this is done only when there is an error, in which case it is okay for that to be slow.